int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -333,6 +333,18 @@ for (var i = 0; i < buttonSelector.length; i++) {
 		buttonSelector[i].addEventListener("mouseover", edqEmailPhoneValidationCallback);
 	}
 }
+/** Added this function to set the listener tothe button on the selected stage (billing/shipping)
+* For more information see Bug #147798 */
```

Comment:
By the way, the convention for JavaScript comments is more like [this](https://typedoc.org/guides/doccomments/)